server/pulp/server/async/app.py

```diff
@@ -87,6 +84,13 @@ def initialize_worker(sender, instance, **kwargs):
         get_resource_manager_lock(sender)
 
 
+@worker_shutdown.connect
+def shutdown_worker(signal, sender):
+    msg = _("Worker '%s' shutdown" % sender.hostname)
```

Comment:
This won't i8ln correctly because when gettext collects the strings it will already be replaced. I make this mistake often myself. Instead what you want is:

`_("Worker '%s' shutdown") % sender.hostname`